docs/data-management/post-session.md

```diff
@@ -163,7 +185,30 @@ As new sessions are collected, the corresponding BIDS structures MUST be saved w
     ``` shell
     datalad save -r -m "add: session <session_id>" sub-001/ses-<session_id>
     ```
+
+    !!! danger "Always double-check that data are annexed and the metadata committed to git"
+
+        Although the [creation of a procedure](preliminary.md#creating-a-datalad-dataset) should ensure data and metadata are added to the appropriate version control (Git or Git-Annex), it is possible that some metadata or data formats are not anticipated, or do not follow the general rules.
+
+        A generally good strategy is to avoid recursion (i.e., do not use the `-r` flag), and leverage Bash's `find` and `xargs` tools.
+        For example, the following command line selects metadata files that should be committed to Git within one session (`pilot020`) and saves them:
+
+        ``` shell
+        find sub-001/ses-pilot020 -name "*.tsv" -or -name "*.json" -or -name "*.bvec" -or -name "*.bval" | xargs datalad save --to-git -m '"add(pilot020): new session metadata"'
+        ```
+
+        Correspondingly, we can store NIfTI data and physiological information:
+
+        ``` shell
+        find sub-001/ses-pilot020 -name "*.nii.gz" -or -name "_eyetrack.tsv.gz" -or -name "_physio.tsv.gz" | xargs datalad save -m '"add(pilot020): new session NIfTI data, eye tracking and physio"'
+        ```
+
+        Please read [*DataLad*'s `save` documentation](http://docs.datalad.org/en/stable/generated/man/datalad-save.html)
+
 - [ ] Push the new data to the remote storage (if your git containing *DataLad* and the Git annex is different from `origin`, e.g., `github`, replace the name below):
     ``` shell
+    datalad push --to ria-storage
     datalad push --to origin
     ```
+
+    !!! danger "Always double-check that data in the annex are uploaded to the RIA store"
```

Comment:
I think it would be useful to add the command below.

Reply:
what do you mean?

Reply:
Is there a method to double check?

Reply:
AFAIK you can clone the dataset somewhere else and test it pulls down the data :)

Also, when doing the push to ria-storage explicitly it should tell you if it failed.